package.json

```diff
@@ -9,8 +9,11 @@
     "build": "next build",
     "start": "next start -p $PORT",
     "build-static": "next build && next export",
-    "test": "jest --coverage",
+    "cy:open": "cypress open",
+    "test:jest": "jest --coverage",
```

Comment:
We could add --watchAll --detectOpenHandles --silent=false to ensure we see the errors for the tests created

Reply:
Would it be better to create coverageThreshold?
https://jestjs.io/docs/configuration#coveragethreshold-object